arcgis-ios-sdk-samples/Layers/Play a KML Tour/README.md

```diff
@@ -0,0 +1,48 @@
+# Play a KML Tour
+
+Play tours in KML files.
+
+![Screenshot](image1.png)
+
+## Use case
+
+KML, the file format used by Google Earth, supports creating tours, which can control the viewpoint of the scene, hide and show content, and play audio. Tours allow you to easily share tours of geographic locations, which can be augmented with rich multimedia. Runtime allows you to consume these tours using a simple API.
+
+## How to use the sample
+
+The sample will load the KMZ file automatically. When a tour is found, the Play button will be enabled. Use the Play and Pause buttons to control the tour. When you're ready to show the tour, use the reset button to return the tour to the unplayed state.
+
+## How it works
+
+1. Load the KML dataset and add it to a layer.
+2. Create the KML tour controller. Wire up the buttons to the `play()`, `pause()`, and `rewind()` methods.
+3. Use a method to explore the tree of KML content and find a KML tour. Once a tour is found, provide it to the KML tour controller.
```

Comment:
`Use a method` is a bit ambiguous as it does not say which method. I'd just remove it.

Reply:
I believe that wording was used since the particular method is platform specific (not provided by the runtime).